tests/unit/db/bigquery/test_bigquery_loader.py

```diff
@@ -1,12 +1,19 @@
 import unittest
 
 from records_mover.db.bigquery.loader import BigQueryLoader
-from records_mover.records.records_format import DelimitedRecordsFormat, ParquetRecordsFormat
+from records_mover.records.records_format import (
```

Comment:
This file, one of the 3 largest in the project, increased in size to 378 lines.  The total size of those files is now 3420 lines (target: 1103).  Is this file complex enough to refactor?